### PR TITLE
Add bond info

### PR DESCRIPTION
**Pull request by Bookcliff**
Added bond information to both the offerings and bonds details pages. Updated bond information so that it varies based on what is available for each bond.

image

- [x] Cleaned code
- [x] Cursory code review

### CODE REVIEW

File: src/pages/BondDetail/bondInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const bondInformation = [
```
Comment (RusseII) [inline src/pages/BondDetail/bondInformation.tsx:1]: beautiful!

Comment (RusseII) [reply src/pages/BondDetail/bondInformation.tsx:1]: you could probably make this whole thing an auction with the key being the bondID, then you wouldn't have to filter

Comment (Bookcliff) [reply src/pages/BondDetail/bondInformation.tsx]: Fixed!

**Comment by Namaskar-1F64F** [comment]
lgtm!

**Pull request merged**